### ISSUE TITLE
get_minutes broken in 12.0.0

### ISSUE THREAD

**Issue by weightwatchers-carlanderson** [hhursev/recipe-scrapers#313]
`get_minutes` somehow broke sometime between 10.1.1 and 12.0.05:

```python
Python 3.7.6 (default, Jan  8 2020, 13:42:34)
[Clang 4.0.1 (tags/RELEASE_401/final)] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from recipe_scrapers._utils import get_minutes
>>> get_minutes('24 Hours')
240
>>> get_minutes('24 hours')
240
>>> get_minutes('24 hrs')
240
>>> import pkg_resources
>>> pkg_resources.get_distribution('recipe_scrapers').version
'12.0.5'
```

and it worked in older versions:

```python
Python 3.7.6 (default, Jan  8 2020, 13:42:34)
[Clang 4.0.1 (tags/RELEASE_401/final)] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from recipe_scrapers._utils import get_minutes
>>> get_minutes('24 hours')
1440
>>> get_minutes('24 hrs')
1440
>>> get_minutes('24 Hours')
1440
>>> import pkg_resources
>>> pkg_resources.get_distribution('recipe_scrapers').version
'10.1.1'
```

**Comment by weightwatchers-carlanderson** [comment]
looks like it broke going from 11.0.0 to 12.0.0

**Comment by hhursev** [comment]
ah weird, thanks for the details - I'm on it!

**Comment by hhursev** [comment]
Fixed in version 12.1.5 @weightwatchers-carlanderson . You can update through pip

**Comment by weightwatchers-carlanderson** [comment]
@hhursev Wow! Confirmed. That is amazingly responsive. Thank you so much
